unit_07/hypothesis_test_practice_activity.Rmd

```diff
@@ -27,51 +44,161 @@ salary_potential   <- read_csv(file = paste0(base_url, 'salary_potential.csv'))
 historical_tuition <- read_csv(file = paste0(base_url, 'historical_tuition.csv'))
 diversity_school   <- read_csv(file = paste0(base_url, 'diversity_school.csv'))
 ```
-```{r}
+```{r show head of tuition data}
 head(tuition_income)
 ```
 
 # One-Sample t-test 
 
 What was the cost of tuition for the 1985-1986 school year, in current dollars, for public schools? 
 
-```{r}
+```{r show tuition cost in 1985}
 ## this query will return a list of the costs for all institution types. 
 historical_tuition %>% 
-  filter(
-    year == '1985-86' & 
-      tuition_type == 'All Current')
+  filter(year == '1985-86' & tuition_type == 'All Current')
+```
 
-## add a filter to this query to return only the rows for public schools 
-## once you're able to add this filter, also add a `select()` verb to your 
-## code (on a new line, after a new pipe) so that saved into the object 
-## public_school_cost_1985 is only `tuition_cost`.  
+Can you directly select the values that are shown in this table? Specifically, suppose that you are interested in the following question: 
 
+> Has the cost of tuition for public schools changed since 1985? 
+
+In order to accomplish this, start from the query written above, but add additional conditions to `filter` on, and also `select` only the columns that you are interested in. 
+
+Once you have formulated your "query", save the resulting values into a numeric object that is called `public_school_cost_1985`. Note that this will require you to convert the type from a data.frame to a numeric value. 
+
+```{r select only public school tuition}
 public_school_cost_1985 <- historical_tuition %>% 
   filter(
-    year == '1985-86' & 
-      tuition_type == 'All Current' & 
-      type == 'Public') %>% 
+    year == '1985-86' & tuition_type == 'All Current' & type == 'Public') %>% 
   select(tuition_cost) %>% 
   as.numeric() # This last line is to store the value outside a data.frame.
+```
 
+```{r test result}
 ## so that you can check your results, the value should be $3571.
+test_that(
+  'public school tuition is $3571', 
+  expect_equal(public_school_cost_1985, 3571)
+)
 ```
 
+Now then, using data that is available to you, please test whether public school tuition has changed from 1985 to the present. In order to do you, you will have to select the appropriate rows and columns of data, and conduct the appropriate test for this data. 
+
+1. What is the appropriate test? 
+2. What are the assumptions for this test? 
+3. Are these assumptions for the test satisfied in this case?
```

Comment:
I worry that this phrasing makes it sound like we're really interested in a final yes or no answer. I prefer the way we did it on the lab, something like "evaluate each assumption, based on your background knowledge, data visualizations, and numerical summaries."